fv3core/grid/mirror.py

```diff
@@ -0,0 +1,342 @@
+import math
+
+from ..utils.global_constants import N_TILES, PI, RADIUS, RIGHT_HAND_GRID
+
+
+__all__ = ["mirror_grid"]
+
+# A tile global version of mirror_grid
+# Closer to the Fortran code
+def global_mirror_grid(grid_global, ng: int, npx: int, npy: int, np):
```

Comment:
Do we still use this code? If we don't, can you either delete it, or (maybe better) add a unit test that the global version gives the same result as the new distributed version?